src/main/java/mods/railcraft/common/blocks/RailcraftBlocks.java

```diff
@@ -206,23 +191,37 @@ public void register() {
             if (!(block instanceof IRailcraftBlock))
                 throw new RuntimeException("Railcraft Blocks must implement IRailcraftBlock");
             IRailcraftBlock blockObject = (IRailcraftBlock) block;
-            blockObject.initializeDefinintion();
-            Railcraft.instance.recipeWaitList.add(blockObject);
```

Comment:
Where did the recipe definitions go?

Reply:
It goes to `RailcraftModulePayload`, in one of the initializing methods.